Documentation updates for ome-zarr comparison mentions (#40)
WayScience/ome-arrow-benchmarks: README.md

> Read this section for a quick summary. The individual benchmark sections below provide the full picture.

> - **OME-Arrow reads faster and writes faster than OME-Zarr.** OA NT Parquet writes 2.6–3.4× faster than OME-Zarr across all tested datasets, and reads 20–50% faster for bulk access. OA NT Vortex is 6–14× faster than OME-Zarr for bulk reads. For random single-image access, OA NT Parquet is slower than Zarr, but OA NT Vortex and Lance are both faster. OA DS (OMEArrowDataset) Parquet is the slowest OME-Arrow variant but remains comparable to OME-Zarr.

Reviewer: what is "bulk access", "bulk reads"? Oh, is this in contrast to "single-image access"? Should bulk access be "multiple images". 

Importantly, why is this important for image-based profiling? What would it mean, at scale? 

Are there other features that make OME-arrow a more suitable choice?

Peripherally, _why_ is it better?

Reviewer: make sure to update the `x`'s here too (see below)

Author: Thanks - yes, that's right for bulk. I added a point about why with image-based profiling. I added a note also about the storage back-end flexibility.

Compares table formats that embed a single OME image column, alongside directory-per-image TIFF and OME-Zarr as baselines.

> OME-Arrow nested tables (Parquet backend) write 2.6–3.4× faster than OME-Zarr and read 20–50% faster for bulk access across all tested datasets.

Reviewer: ```suggestion
OME-Arrow nested tables (Parquet backend) write 2.6–3.4x faster than OME-Zarr and read 20–50% faster for bulk access across all tested datasets.
```

Author: Thanks, addressing these.

> OA NT Vortex extends that read advantage to 6–14×.

Reviewer: ```suggestion
OA NT Vortex extends that read advantage to 6–14x.
```

Author: Thanks, addressing these.

**How to read this figure:** the bars are grouped by dataset (x-axis) and colored by format. Compare formats within each dataset group to understand relative trade-offs; compare across dataset groups to check whether the pattern holds for different image types and sizes.

> OA NT writes faster than OME-Zarr (~3× on NF1) and reads faster for bulk access (~1.5×); OA NT Vortex extends the read advantage to ~12×.

Reviewer: ```suggestion
OA NT writes faster than OME-Zarr (~3x on NF1) and reads faster for bulk access (~1.5x); OA NT Vortex extends the read advantage to ~12x.
```

These `x` values looked wonky to me

Author: Thanks, addressing these and others.
